lib/hydra/pcdm/models/concerns/object_behavior.rb

```diff
@@ -77,6 +67,24 @@ def parent_collections
       aggregated_by.select { |parent| parent.class.included_modules.include?(Hydra::PCDM::CollectionBehavior) }
     end
 
+    def child_objects= objects
+      raise ArgumentError, "each object must be a pcdm object" unless objects.all? { |o| Hydra::PCDM.object? o }
+      raise ArgumentError, "an object can't be an ancestor of itself" if object_ancestor?(objects)
+      self.members = objects
+    end
+    def objects= objects
```

Comment:
Please add a blank line above this method.  Please add some test coverage for this method. Presently it has none.